salt/metalk8s/saltmaster/files/minion_99-metalk8s.conf.j2

```diff
@@ -0,0 +1,5 @@
+master: localhost
```

Comment:
Hmm this one should not be localhost we should put the master ip.
But for the moment we don't have anywhere where you can store it. Should be in the pillar but as we have nothing to generate it for the moment ...

At least be able to configure it for the moment even if we default to `localhost`

Reply:
Or just add a new ticket ;)